examples/hello-world-service/hello_world_service.md

```diff
@@ -1,10 +1,14 @@
 # Hello world service
 
-Let's write a simple HTTP service in Ballerina. 
-This example demonstrates the network primitives in the language that make it simpler to develop services.
+Let's write a simple HTTP service in Ballerina. This example demonstrates the network primitives in the language that make it simpler to develop services.
 
 ::: code hello_world_service.bal :::
 
-::: out hello_world_service.client.out :::
+Execute the command below to run the above service.
```

Comment:
What if we simply say `Run the service as follows`?

Reply:
Updated.